examples/wasm_ledger/test/test.ledger.js

```diff
@@ -20,7 +20,7 @@ describe("LEDGER TEST", function () {
   var sim,
       transport;
 
-  beforeEach(async function() {
+  before(async function() {
```

Comment:
as discussed, let's revert this back to beforeEach
We can open an issue so Zemu does not clean the screen on each start.